rudder-lang/src/technique.rs

```diff
@@ -0,0 +1,338 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
main changes, rewritten most of translate.rs to create a serde Technique representation of json technique that have only to_rudderlang and to_json -> String. And utility functions